openmdao/drivers/pyoptsparse_driver.py

```diff
@@ -389,6 +389,9 @@ def run(self):
             # optimizers other than pySNOPT may not populate this dict
             pass
 
+        self.opt_prob = None
```

Comment:
It is possible we don't even need to save either of these in the instance in the first place.

Reply:
Nothing seemed to be referencing them, so I went ahead and removed them.  Didn't break anything as far as I can tell.